frontend/packages/dev-console/src/components/import/section/ResourceSection.tsx

```diff
@@ -23,10 +23,10 @@ type ResourceSectionProps = {
 const createHelpText = (k8sModel: K8sKind, helpText: string) => {
   return (
     <>
-      <p>
+      <div style={{ paddingBottom: 'var(--pf-global--spacer--xs)' }}>
```

Comment:
lol - use your SCSS file. Inline should be avoided in all cases possible.

Reply:
Yes I can do that but here it doesn't seem necessary because it is only a single style, also this is only targeted at this and is not used anywhere else. Inline styles are being used in a lot of places in console as well.

Reply:
I add inline css when no more than one style is required and it is not going to be used anywhere else. In this case having a inline css or a class seemed to have equal weight. Updated anyways.

Reply:
I don't agree; inline styles are a poor way to write maintainable code. If UX came back and asked us to add more styles (colour the resource name, for instance), you don't have a nice spot to put them, you have to refactor your styles into a css file and then add the new styles (or just clutter up the React code some more). Static styles (imo) have no purpose being written inline.

It's convenient to use the `style` attribute, I understand that, but we are not writing code that we're going to toss out. We're writing code we'll have to maintain; best write it in such a fashion that we don't have to rewrite for small additions.